frame/staking-rewards/README.md

```diff
@@ -1,116 +1,221 @@
+    - [Routing](#routing)
+  - [Positions](#positions)
+    - [Staking](#staking)
+    - [Staked asset](#staked-asset)
+    - [Unstake](#unstake)
+    - [Unstake while locked](#unstake-while-locked)
+    - [Claiming](#claiming)
+    - [Split position](#split-position)
+    - [Extend amount](#extend-amount)
+    - [Extend time](#extend-time)
+    - [Expiration](#expiration)
+    - [Compounding](#compounding)
+  - [Notes](#notes)
+  - [References](#references)
 
-### Configuring an asset as being rewardable
+## General flow
 
-In order to allow the users to stake an asset, we first need to configure it. This
-is done through the `configure` extrinsic, allowing us to provide staking
-configurations such as:
+After paying an existential deposit, any account can create a `staking pool`.
 
-- the set of staking durations along their reward multiplier (e.g. `[(WEEK,
-  0.5), (MONTH, 0.8), (TWOMONTH, 1.0)]`)
-- the list of assets we can reward the stakers with (e.g. `[PICA, BTC, ETH]`)
-- the early unstake penalty, applied on the staked asset when the user unstake
-  early (before the end of the selected staking duration)
-- the penalty beneficiary, the account where the penalty are going to be
-  transferred to
+The staking pool has a configuration explaining which staking positions other accounts can have.
 
-Once this extrinsic has been dispatched, let's assume we want to reward `PICA`
-stakers, the mapping will be `PICA => ([(WEEK, 0.5), (MONTH, 0.8), (TWOMONTH,
-1.0)], [PICA, BTC, ETH])`. Meaning that users will be able to stake `PICA` for
-either a `WEEK` for a reward multiplier of `0.5` or a `MONTH` etc... The
-incentive for the users to stake will be the fact that they will be able to
-harvest `PICA` or `BTC` or `ETH` whenever a reward distribution occured.
+User transfers staked assets into the staking pool and choose what position he wants to take. She becomes a `staker`.
+
+Protocol owner increases total rewards share as time goes.
+
+Anybody can mint and transfer rewarded assets into pool. This process can be automated.
+
+A `staker` may leave a position, claim rewards, stake more, etc.
+Or she can make [a position to be fNFT](../nft/README.md) and use it as an instrument in other protocols.
+
+
+## Protocol
+
+The `staking protocol` is configured with ED for having pools and positions on-chain.  
+
+## Pool
+
+A pool is where users put tokens and get shares of rewards or other powers. 
+
+Any protocol can have one or more pools. The main characteristic of a pool is staked asset identifier(token).
+
+Each pool is governed through a protocol that created it.
+
+### Configuration
+
+To allow the users to stake an asset, we first need to `configure` the pool. 
+
+Configuration of a pool defines:
+
+Can/must user time lock tokens to get, possibly increased, `share`. What is the penalty for early unlock if there is a lock
+  
+If share decays with time until lock ends and extending time starts from zero elapsed.
```

Comment:
Let's just remove decay stuff, not a requirement

Reply:
fixing in next PR.